ui/src/stat_table.tsx

```diff
@@ -135,10 +137,10 @@ export const
           header = columns.map((label, i) => (
             <th key={`${i}:${label}`}>{label}</th>
           )),
-          rows = items.map(({ name: rowName, label, values, caption, icon, icon_color }, i) => {
+          rows = items.map(({ name: rowName, label, values, caption, icon, icon_color, value_colors: colors }, i) => {
             const
               onClick = rowName ? () => jump(tableName, rowName) : undefined,
-              cells = values.map((value, j) => (<td key={`${j}:${value}`}>{value}</td>))
+              cells = values.map((value, j) => (<td key={`${j}:${value}`} style={colors && colors[j] ? { color: colors[j] } : undefined }>{value}</td>))
```

Comment:
[Spectrum colors](https://github.com/h2oai/wave/blob/master/ui/src/index.scss#L63-L81) should be supported as well. Please check if these work:

* spectrum colors
* regular CSS colors - hex / RGB / browser colors - red/blue/green

```suggestion
              cells = values.map((value, j) => <td key={`${j}:${value}`} style={colors && colors[j] ? { color: cssVar(colors[j]) } : undefined }>{value}</td>)
```